packages/clerk-js/src/core/clerk.ts

```diff
@@ -1526,6 +1526,17 @@ export class Clerk implements ClerkInterface {
     return this.navigate(to);
   }
 
+  /**
+   * This clears the emotion cache, which is particularly useful
+   * when working with Astro View Transitions to prevent style loss during page navigation.
+   *
+   * For more info on the issue, see:
+   * https://github.com/withastro/astro/issues/11000
+   */
+  __internal_flushEmotionCache() {
```

Comment:
@nikosdouvlis asking in advance here if there's a better way to access the internal emotion cache rather than what I did here. Any advice would do!

Need it to make view transitions work in Astro. I got the idea [here](https://ntsim.uk/posts/how-to-update-or-remove-global-styles-in-emotion#the-aha-moment)

Reply:
Yep, this adds 140KB to the [bundle size](https://ja2r7.app.goo.gl/HJZmjBwm6LFCHgpf9) so this is not a good solution